### DISCUSSION TITLE
Intellij Plugin not available in the latest EAP

### DISCUSSION THREAD

**Discussion by ennishol** [jbangdev/jbang#1760] [Q&A] [answered]
Hi, I'm using the latest EAP of the IDE IntelliJ IDEA 2024.1 EAP (Ultimate Edition) and the plugin is not available for installation. Is it not yet supported? Is there any workaround? 
Thanks!

```
Build #IU-241.14024.14, built on February 28, 2024
Expiration date: March 29, 2024
Runtime version: 17.0.10+8-b1207.1 amd64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o.
Linux 5.15.133.1-microsoft-standard-wsl2
```

**Comment by ennishol** [comment]
Tried to install from disk. Got
![image](https://github.com/jbangdev/jbang/assets/64908285/4328f5a1-f057-4d89-b169-c8a359820854)

**Comment by maxandersen** [comment]
best to open issue in @jbangdev/jbang-idea - I'm not sure why EAP is not matching.

**Comment by maxandersen** [comment]
maybe @linux-china knows?

**Reply by linux-china** [reply]
@maxandersen  I have upgraded the code and compatible with IntelliJ IDEA 2024.1, but  plugin published failed.   Could you take a look at the GitHub action for publish?

**Reply by maxandersen** [reply, marked as answer]
Fixed last night. Should be published now.

**Comment by ennishol** [comment]
Thanks! Now the installation worked.
